app/code/Magento/Sales/Block/Order/OrderItemsPager.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Sales\Block\Order;
+
+use Magento\Framework\Data\Collection;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Html pager block
+ *
+ * @SuppressWarnings(PHPMD.ExcessivePublicCount)
+ * @api
```

Comment:
Please remove @api tag, we will have to mark this as @api on minor release.

Reply:
Has been removed.